### PR TITLE
#568 Synchronized the project description 

### PR DESCRIPTION
**Pull request by HarryChiang143**
'Easy to deploy the streaming application'

### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #568)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

required by #568

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Ohara Stream
 
-An easy-to-use visual stream processing tool based on Apache Kafka.
+Easy to deploy the streaming application
```
Comment (chia7712) [inline README.md]: Easy to deploy various types of streaming applications.

I prefer to highlight the word -- "various"

Comment (vitojeng) [reply README.md]: Can you explain the meaning of "various types"?

Comment (chia7712) [reply README.md]: > Can you explain the meaning of "various types"?

因為我們的強項就是支援"各式各樣的streaming application"，此外如果你們去看各個系統的廣告詞，大多都會用"多數""複數"來形容自己所支援的東西，就像中文常常會說"應有盡有"

不過昨天愷說他不想改，畢竟只是一段敘述，我就直接合併了

Comment (vitojeng) [reply README.md]: @chia7712 ok. thanks for the explanation!

Comment (HarryChiang143) [reply README.md]: I think @chia7712  made a good suggestion, maybe the next description change we can take it

**Pull request merged**